playbook-website/app/javascript/components/VisualGuidelines/Examples/ZIndex.tsx

```diff
@@ -24,9 +24,21 @@ const TOKENS = {
   '$z_10': 1000,
 }
 
+const screenSizeProps = {
+  display: ['xs', 'sm', 'md', 'lg', 'xl']
+}
+
+const UTILITY_CLASSES = [
+  {size: 'xs', media: '@media screen and (max-width: 575px)', class: '.z_index_xs_{1-10}', properties: 'z_index: {100-1000} !important'},
+  {size: 'sm', media: '@media screen and (max-width: 576px)', class: '.z_index_sm_{1-10}', properties: 'z_index: {100-1000} !important'},
+  {size: 'md', media: '@media screen and (max-width: 768px)', class: '.z_index_md_{1-10}', properties: 'z_index: {100-1000} !important'},
+  {size: 'lg', media: '@media screen and (max-width: 992px)', class: '.z_index_lg_{1-10}', properties: 'z_index: {100-1000} !important'},
+  {size: 'xl', media: '@media screen and (max-width: 1200px)', class: '.z_index_xl_{1-10}', properties: 'z_index: {100-1000} !important'},
```

Comment:
Lets use the tokens instead of raw values?


We export them for use [in JS files here](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/tokens/exports/_screen_sizes.scss)
You can see them being [used in this file here](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_dashboard/commonSettings.js)